webapp/src/app/components/dialogs/new-map/new-map.component.scss

```diff
@@ -0,0 +1,40 @@
+@import "~@angular/material/theming";
```

Comment:
styles are not needed in this component. All it does is display an overlay container with the map settings. Both of them have their own styles